documentation/style-guide.rst

```diff
@@ -196,6 +199,35 @@ language):
     achieve the same effect.  This assures that files are flushed and file
     descriptor resources are released in a timely manner.
 
+
+Author attribution
+==================
+
+For new documentation, do not use a byline (naming the author of the document).
+Explicit attribution tends to discourage other users from updating community
+documentation.
+
+Existing documentation with bylines will not be changed unless the author
+decides to do so. This is subject to change in the future.
+
+
+First-person language
+=====================
+
+Avoid first-person language.  Our documentation is owned and maintained by a
+community.  First-person language mistakenly implies a single author.
```

Comment:
Would it be nicer to phrase this positively, saying something like:

```suggestion
Grammatical person 
==================

We try to maintain a consistent tone of voice across our documentation.
As part of this, prefer using the third-person when writing reference text and either the second or third person when writing tutorials or guides, which are naturally more 'interactive'.
```

This removes the negative language from the text, and additionally provides better guide rails for authors of different types of text.

Reply:
Should a note be added about existing documents that use the first-person?
Adding a few examples would be useful too.
Is the first-person plural allowed?  I've seen it used (and used it myself) in tutorials and howtos.

Reply:
I like the original language here better than @AA-Turner's suggestion; it is more concise and clear.

Reply:
My concrete suggestion was a very quick draft -- the point was more about using constructive language and adding more guidance. Just to clarify that the suggestion is more an opening offer than final wording!

Reply:
Style guides often tell you what to avoid. A quick example:

https://developers.google.com/style/inclusive-documentation

Reply:
I like the idea of providing examples of good and bad usage.  Also, first-person plural can be useful: "If we change the method like this, ..."

I'll work on a re-wording tomorrow.

Reply:
I've done some more reading and thinking, and I'd like to remove this from this pull request and have more discussion before making a recommendation.

Reply:
BTW, here is a pull request that removes some first-person language: https://github.com/python/cpython/pull/107449/files, with discussion.